File: 1-two-sum/two-sum.js
const twoSum = (nums, target) => {
    const obj = {};
    for (let i = 0; i < nums.length; i++) {
        let n = nums[i];
        if (obj[target - n]>=0) {
            return [obj[target - n], i];
        } else obj[n] = i;
    }
};

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict={}
        for index in range(len(nums)):
            currValue=nums[index]
            diff=target-currValue
            if diff in dict:
                return [dict[diff],index]
            else: dict[currValue]=index    
